cmd/opflexagentcni/main.go

```diff
@@ -99,6 +100,10 @@ func loadConf(args *skel.CmdArgs) (*NetConf, *K8SArgs, string, error) {
 	id := args.ContainerID
 	log.Debug("Args: ", k8sArgs)
 
+	if n.WaitForNetwork && n.WaitForNetworkDuration == 0 {
+		n.WaitForNetworkDuration = 210
+	}
+
```

Comment:
perhaps we can also add a debug log here which logs n.WaitForNetworkDuration final value